### DISCUSSION TITLE
DHCP Option : How-to ?

### DISCUSSION THREAD

**Discussion by samsam-rolon** [AdguardTeam/AdGuardHome#2941] [Q&A]
Hello
My ISP required somme custom option in the DHCP server in order to run my TV-BOX. 
How can I do ?
Here my DHCPD Config : 
``` conf
interface=ovs_bond0
dhcp-option=tag:tv,option:router,192.168.1.1
dhcp-option=tag:tv,option:domain-name,home
dhcp-option=tag:tv,option:dns-server,192.168.1.1
dhcp-option-force=tag:tv,option:ntp-server,172.20.224.167
dhcp-option=tag:tv,72,193.253.67.89
dhcp-option-force=tag:tv,vi-encap:3561,4,"12345Z"
dhcp-option-force=tag:tv,vi-encap:3561,5,"AA12345ZZ678900"
dhcp-option-force=tag:tv,vi-encap:3561,6,"Livebox Fibre"
dhcp-boot=tag:tv,,192.168.1.1
dhcp-host=5C:5C:5C:5C:5C:5C,set:tv,LB-TVUHD,192.168.1.1,86400
```

**Comment by ameshkov** [comment]
@ainar-g can we maybe come up with an instruction on how to convert this to AdGuard Home config?

**Reply by ainar-g** [reply]
I could probably write an article about it (or a section in the current wiki article about DHCP), but probably not during this cycle.

**Reply by samsam-rolon** [reply]
Nice. In an other way, any way to "guess" how to do ? Like where is the function/file witch parse it ?

**Reply by ainar-g** [reply]
@samsam-rolon, you can read through [this RFC] to get the numbers of the options.  Option descriptions start at page 4.

[this RFC]: https://tools.ietf.org/html/rfc2132

**Reply by samsam-rolon** [reply]
Okay, I make it works but no. In wireshark, I doesn't saw the customisation... #3101 
Moreover, the setting looks like "remove" on the YAML each time i'm browsing option in GUI
